lib/runners/go.js

```diff
@@ -53,6 +53,8 @@ func Test${name}(t *testing.T) {
         // exit status 1
         // FAIL	{dirname of test file}	{seconds}s
         sanitizeStdOut: function(stdout) {
+            // special case when build failed
+            if (stdout.includes('[build failed]')) return '<ERROR::>Build Failed\n';
```

Comment:
I think I omitted it because it gave out the answer, but I'll look into it and let you know.